README.md

```diff
@@ -2,32 +2,67 @@
 
 Guiguts rewrite using Python/tkinter
  
- 
-## Windows-focused dev installation
+## Windows Development Setup
 
 Minor alterations probably needed for other platforms
 
-## Install Python
+### Install Python
 
-    1. Download 3.10 from python.org
-    2. Install – default dir is `C:\Users\<username>\AppData\Local\Programs\Python\Python310`
-    3. Ensure this dir is in PATH variable
+1. Download 3.10 from [python.org](https://www.python.org/).
+2. Install – default dir is `C:\Users\<username>\AppData\Local\Programs\Python\Python310`
+3. Ensure this dir is in PATH variable
 
-## Install Poetry
+### Install Poetry
 
 1. https://python-poetry.org/docs/#installing-with-the-official-installer 
 2. `(Invoke-WebRequest -Uri https://install.python-poetry.org -UseBasicParsing).Content | py -`
 3. Installs into `C:\Users\<username>\AppData\Roaming`
 4. Adds top-level “poetry” wrapper in `C:\Users\<username>\AppData\Roaming\Python\Scripts`
 5. Ensure the latter is in PATH variable
 
-## Set up Development System
+## macOS Development Setup
+
+Developing Guiguts on macOS requires installing [Homebrew](https://brew.sh/) first.
+
+### Install Python
+
+Install python and python-tk using Homebrew. Note that we install and use a
+specific version in the commands below for consistency with other developers.
+
+```bash
+brew install python@3.10 python-tk@3.10
+```
+
+### Install Poetry
+
+We also install poetry using Homebrew.
+
+```bash
+brew install poetry
+```
+
+## Common Development Setup
 
 1. Install Python & Poetry (above)
 2. Clone the [GG2 Github repo](https://github.com/windymilla/Guiguts-2.0-tkinter)
-3. In the cloned GG2 directory, `poetry install`. The message
-   ".../guiguts does not contain any element" can be ignored (or suppressed by using
-   `poetry install --no-root` instead).
+   or a fork thereof.
+3. In the cloned GG2 directory, create a virtual environment using the version of
+   python you installed above.
+   * Windows
+     ```
+     poetry env use C:\Users\<username>\AppData\Local\Programs\Python\Python310\python3
```

Comment:
If this is executed within a bash shell of some sort on Windows, it should probably use `/`s. And there's very likely some env variable that could be used to shorten the user's home directory and wouldn't require them updating `<username>` but I don't have a clue what that is.

Reply:
In the Git Bash shell, the necessary command is
` poetry env use ~/AppData/Local/Programs/Python/Python310/python.exe`
Note that `python3` doesn't exist, and it fails without the `.exe`

Reply:
Thanks - I suspected my wild-ass guess was going to be wrong 😁